packages/valory/customs/kelly_criterion_no_conf/kelly_criterion_no_conf.py

```diff
@@ -139,7 +159,11 @@ def get_bet_amount_kelly(  # pylint: disable=too-many-arguments
 
     info.append(f"Kelly bet amount: {wei_to_native(kelly_bet_amount)} xDAI")
     info.append(f"Bet kelly fraction: {bet_kelly_fraction}")
+    info.append("Applying STATIC kelly fraction to all bets")
     adj_kelly_bet_amount = int(kelly_bet_amount * bet_kelly_fraction)
+    # adj_kelly_bet_amount = get_adjusted_kelly_amount(
+    #     info, kelly_bet_amount, weighted_accuracy, bet_kelly_fraction
+    # )
```

Comment:
Why static?

Reply:
I want to test first the trader with the new tool selection using a static kelly fraction, then collect results and then activate the dynamic kelly fraction, collect results and finally compare with the previous static results